microsoft-edge/extensions-chromium/store-policies/developer-policies.md

```diff
@@ -159,218 +235,325 @@ If you publish a person's Personal Information to an outside service or third-pa
+
+<!-- ------------------------------ -->
+#### 1.10 Advertising conduct and content
 
 For all advertising related activities, the following requirements apply:
 
-#### 1.10.1 Purpose
 
-The primary content of your extension must not be advertising, and advertising must be clearly distinguishable from other content in your extension.
+###### 1.10.1 Purpose
+
+The primary content of your extension must not be advertising, and your extension should not bundle other extension offers within the same flow.
+
+Users should not be misled to click on advertisements to fully use the extension.
 
-#### 1.10.2 Policies and Agreements
+
+###### 1.10.2 Policies and agreements
 
 Any advertising content your extension displays must adhere to [Microsoft Advertising Network policies](https://about.ads.microsoft.com/policies/home).
 
 If your extension displays ads, all content displayed must conform to the advertising requirements of the [App Developer Agreement](/legal/windows/agreements/app-developer-agreement) and this Policy.
 
-#### 1.10.3 Quality of advertising
 
-*   The primary purpose of your extension must not be to get users to click ads.
+###### 1.10.3 Quality of advertising
+
+* The primary purpose of your extension must not be to get users to click ads.
 
-*   Your extension must not do anything that interferes with or diminishes the visibility, value, or quality of any ads that it does display.
+* Your extension must not do anything that interferes with or diminishes the visibility, value, or quality of any ads that it does display.
 
-#### 1.10.4 Promotions
+
+###### 1.10.4 Promotions
 
 If you purchase or create promotional ad campaigns to promote your extensions through the ad campaign functionality in [Partner Center](https://partner.microsoft.com/dashboard/microsoftedge/public/login?ref=dd), all ad materials you provide to Microsoft, including any associated landing pages, must comply with [Microsoft Creative Specifications Policy](https://about.ads.microsoft.com/policies/creative-specs) and [Microsoft Advertising Network policies](https://about.ads.microsoft.com/policies/home).
 
-#### 1.10.5 Notifying Users of Opt-Out for Interest-Based Advertising
+
+###### 1.10.5 Notifying users of opt-out for interest-based advertising
 
 Your privacy statement or terms of use must let users know you plan to send Personal Information to the ad service provider and must tell users how they may opt-out of interest-based advertising.
 
-#### 1.10.6 Other guidelines
+
+###### 1.10.6 Other guidelines
 
 If your extension is directed at children under the age of 13, as defined in the [Children's Online Privacy Protection Act](https://www.ftc.gov/tips-advice/business-center/privacy-and-security/children%27s-privacy); you must notify Microsoft of this fact in [Partner Center](https://partner.microsoft.com/dashboard/microsoftedge/public/login?ref=dd) and ensure that all ad content displayed in your extension is appropriate for children under the age of 13.
```

Comment:
```suggestion
If your extension is directed at children under the age of 13, as defined in the [Children's Online Privacy Protection Act](https://www.ftc.gov/tips-advice/business-center/privacy-and-security/children%27s-privacy); you must notify Microsoft of this fact in [Partner Center](https://partner.microsoft.com/dashboard/microsoftedge/public/login?ref=dd) and ensure that all ad content displayed in your extension is appropriate for children under the age of 13.
```
"Partner Center" occurs 5 times in this article.
4 times are linked - that's excessive linking; linking every mention for no reason.  Recommend un-linking some of the 4 instances.

And they are not a short Partner Center URL, they are a link that took a long time to resolve for me, and then rejected my account:
https://partner.microsoft.com/dashboard/microsoftedge/public/login?ref=dd

Reply:
unlinked